src/util/misc/jszip-helper.ts

```diff
@@ -51,8 +50,8 @@ export async function addFolderToZipRecursively(path: string, zip: JsZip): Promi
     if (subEntityStats.isDirectory()) {
       await addFolderToZipRecursively(subEntityPath, folderZip);
     } else {
-      const fileStream = Fs.createReadStream(subEntityPath);
-      folderZip.file(subEntityName, fileStream);
+      const fileBuffer = await Pfs.readFile(subEntityPath);
```

Comment:
This changes the logic so that it reads the entire file into memory before zipping. This limits the size of files we can add to available RAM. Don't do this unless there's a really good reason.

Reply:
fixed